indra/newview/skins/default/xui/en/notifications.xml

```diff
@@ -3031,6 +3031,14 @@ This is usually a temporary failure. Please customize and save the wearable agai
   <tag>fail</tag>
   </notification>
 
+  <notification
+   icon="notifytip.tga"
+   name="OutOfDiskSpace"
+   type="notifytip">
+The system is out of disk space. You will need to free up some space on your computer or clear the cache.
+<tag>fail</tag>
+  </notification>
```

Comment:
suggestion: mark as unique

Reply:
How about making the "should never be called" base class method pure virtual?

Reply:
I don't have a strong feeling about the wording, think it's OK as-is.